crates/router/src/connector/adyen/transformers.rs

```diff
@@ -4115,6 +4117,13 @@ pub fn is_chargeback_event(event_code: &WebhookEventCode) -> bool {
     )
 }
 
+pub fn is_payout_event(event_code: &WebhookEventCode) -> bool {
+    matches!(
+        event_code,
+        WebhookEventCode::PayoutThirdparty | WebhookEventCode::PayoutDecline
```

Comment:
Are we not handling PAYOUT_REVERSED and PAYOUT_EXPIRE?

ref - https://docs.adyen.com/online-payments/online-payouts/payout-webhook/

Reply:
Added support for them